stream_alert_cli/lookup_tables/handler.py

```diff
@@ -0,0 +1,187 @@
+        cls._setup_get_subparser(lookup_tables_subparsers)
+        cls._setup_set_subparser(lookup_tables_subparsers)
+
+    @classmethod
+    def _setup_describe_tables_subparser(cls, subparsers):
+        generate_subparser(
+            subparsers,
+            'describe-tables',
+            description='Show tables',
+            subcommand=True
+        )
+
+    @classmethod
+    def _setup_get_subparser(cls, subparsers):
+        """
+        Get subcommand:
+
+        $ ./manage.py lookup-tables get -t [table] -k [key]
+        """
+        get_parser = generate_subparser(
+            subparsers,
+            'get',
+            description='Validate defined log schemas using integration test files',
+            subcommand=True
+        )
+
+        get_parser.add_argument(
+            '-t',
+            '--table',
+            help='Name of the LookupTable',
+            required=True,
+            default=None
+        )
+
+        get_parser.add_argument(
+            '-k',
+            '--key',
+            help='Key to fetch on the LookupTable',
+            required=True,
+            default=None
+        )
+
+    @classmethod
+    def _setup_set_subparser(cls, subparsers):
+        """
+        Set subcommand:
+
+        $ ./manage.py lookup-tables set -t [table] -k [key] -v [value]
+        """
+        set_parser = generate_subparser(
+            subparsers,
+            'set',
+            description='Validate defined log schemas using integration test files',
+            subcommand=True
+        )
+
+        set_parser.add_argument(
+            '-t',
+            '--table',
+            help='Name of the LookupTable',
+            required=True,
+            default=None
+        )
+
+        set_parser.add_argument(
+            '-k',
+            '--key',
+            help='Key to set on the LookupTable',
+            required=True,
+            default=None
+        )
+
+        set_parser.add_argument(
+            '-v',
+            '--value',
+            help='Value to save into LookupTable',
+            required=True,
+            default=None
+        )
+
+    @classmethod
+    def handler(cls, options, config):
+        subcommand = options.subcommand
+        if subcommand == 'describe-tables':
+            return cls._describe_tables_handler(config)
+
+        if subcommand == 'get':
+            return cls._get_handler(options, config)
+
+        if subcommand == 'set':
+            return cls._set_handler(options, config)
+
+        LOGGER.error('Unhandled lookup-tables subcommand %s', subcommand)
+
+    # pylint: disable=protected-access
+    @staticmethod
+    def _describe_tables_handler(config):
+        LOGGER.info('==== LookupTables; Describe Tables ====\n')
+
+        lookup_tables = LookupTables.get_instance(config=config)
+
+        LOGGER.info('%d Tables:\n', len(lookup_tables._tables))
+        for _, table in lookup_tables._tables.items():
```

Comment:
you're welcome to use `print` statements when providing feedback to the user if you'd like (similar to what we do for output during tests). instead you'll have everything prefixed with logger stuff. your call really

Reply:
as an aside, it's also much easier to ~handle~ control formatting, etc with print = str.format than with logger output